apps-rendering/src/components/metadata.tsx

```diff
@@ -97,6 +97,11 @@ const liveBlogPadding = css`
 const toggleOverrideStyles = css`
 	padding-right: ${remSpace[1]};
 
+	button {
```

Comment:
This unrelated change fixes a styling regression in the AR `metadata` component which was introduced by removing the margin in the Source `ToggleSwitch` component here: https://github.com/guardian/source/pull/1255